test/test_filter_hanarp_message.rb

```diff
@@ -62,20 +63,21 @@ def test_filter_missing_machine_id
             { 
                 "message" => ": 2018 Feb  9 21:07:41 GMT: %UCSM-6-EVENT: [] [FSM:BEGIN]: Soft shutdown of server sys/chassis-4/blade-7",
                 "SyslogSource" => "1.1.1.1",
-                "event" => "Soft Shutdown",
-                "stage" => "begin"
+                "event" => "Restart",
+                "stage" => "end"
             }
         ]
         filtered_records = filter(messages)
         data = JSON.parse(filtered_records[0]['message'])
         
-        assert_equal "Soft Shutdown", data['event']
+        assert_equal "Restart", data['event']
```

Comment:
Do you have a test for evaluating an Unknown case? If not, add one.

Reply:
done